espflash/src/cli/mod.rs

```diff
@@ -106,6 +106,9 @@ pub struct FlashArgs {
     /// Path to a CSV file containing partition table
     #[arg(long, value_name = "FILE")]
     pub partition_table: Option<PathBuf>,
+    /// Label of target app partition
+    #[arg(long, value_name = "LABEL")]
+    pub target_app_partition: Option<String>,
```

Comment:
Do you mind moving these arguments to `espflash/src/bin/espflash.rs` and `cargo-espflash/main.rs`? It would require some duplication but, changing the CLI mod would result in a breaking change requiring us to bump the version to `3.0`